lib/r10k/action/deploy/module.rb

```diff
@@ -28,7 +28,7 @@ def initialize(opts, argv, settings)
                 generate_types: @generate_types
               },
               modules: {
-                requested_modules: @argv,
+                requested_modules: @argv.map.to_a,
```

Comment:
This was a bug in that was exposed with further functional testing. `@argv` here _acts_ like an array but is actually a CRI::ArgumentList that includes the Enumerable module. This causes problems with logging that calls `inspect` on the value. This change was accidentally rolled into another refactor, but I can break it out if that's best.